### ISSUE TITLE
Tracks should be labeled by their labels

### ISSUE THREAD

**Issue by christoph-fricke** [holisticon/mutti#3]
In the POC, `mutti-track`s filtered their children for a `mutti-label` and assigned them a ID for `aria-labelledby`.

This should either be included as well or replaced with a better approach. As a result, a browser should correctly label a track based in its inner label element.
